editor/sidebar/post-author/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { PanelRow, withInstanceId } from 'components';
+import { Component } from 'element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEditedPostAttribute } from '../../selectors';
+import { editPost } from '../../actions';
+
+class PostAuthor extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			authors: [],
+		};
+	}
+
+	fetchAuthors() {
+		this.fetchAuthorsRequest = new wp.api.collections.Users().fetch( { data: {
+			roles: 'author,editor,administrator',
+			per_page: 100,
+		} } );
+		this.fetchAuthorsRequest.then( ( authors ) => {
+			this.setState( { authors } );
+		} );
+	}
+
+	componentDidMount() {
+		this.fetchAuthors();
+	}
+
+	componentWillUnmount() {
+		if ( this.fetchAuthorsRequest ) {
```

Comment:
I notice in some cases (latest posts, for example) we're checking the status as pending before deciding to abort. If the object let's us call `abort` anyways, and this property is always defined, maybe we don't need the condition at all?